ios/objective-c/AmpliObjectiveCSampleApp/AmpliObjectiveCSampleApp/Ampli/Ampli.m

```diff
@@ -18,7 +18,7 @@
 #import "Amplitude.h"
 #import "AMPPlan.h"
 
-@implementation Event
+@implementation Event: NSObject
```

Comment:
I think Event might need to extend NSObject

Reply:
Fixed